src/dlgTriggerEditor.cpp

```diff
@@ -3128,8 +3161,8 @@ void dlgTriggerEditor::saveTrigger()
         else if( _type == 5 ) regexPropertyList << REGEX_LINE_SPACER;
         else if( _type == 6 ) regexPropertyList << REGEX_COLOR_PATTERN;
     }
-    QString script = mpSourceEditor->toPlainText();
 
+    QString script = mpSourceEditorEdbeeDocument->text();
```

Comment:
Edbee does not support "rich-text" does it?  We don't want HTML tags in the text we get back from the editor which is why we had to use `QTextEdi::toPlainText()` when we used the Qt editor widget...